### DISCUSSION TITLE
在仿真easyrl中DQN的代码时出错

### DISCUSSION THREAD

**Discussion by yxz777** [datawhalechina/DOPMC#170] [问题求助] [answered]
([-0.0360446  -0.01908354 -0.00314136 -0.04963698], {})在第一个状态时，state总会多一个{}，请问这是什么情况已经如何解决呢？

**Comment by johnjim0816** [comment, marked as answer]
> ([-0.0360446 -0.01908354 -0.00314136 -0.04963698], {})在第一个状态时，state总会多一个{}，请问这是什么情况已经如何解决呢？

gym版本不对，请按照readme安装
